<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1636911671956' stop='1636911709743'>
    <name>Backbase Assessment</name>
    <title>Backbase Assessment</title>
    <test-cases>
        <test-case start='1636911671956' status='passed' stop='1636911675728'>
            <name>Create New Article</name>
            <title>Create New Article</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1636911675728' status='failed' stop='1636911709742'>
            <name>Update New Article</name>
            <title>Update New Article</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4281:23)
                    at listOnTimeout (node:internal/timers:557:17)
                    at processTimers (node:internal/timers:500:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>